src/scripts/main.js

```diff
@@ -1,3 +1,342 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const headers = table.querySelectorAll('th');
+const tableBody = table.querySelector('tbody');
+
+headers.forEach(function(header) {
+  header.dataset.sorted = 'DESC';
+});
+
+function sortColumn(index, isASC) {
+  let sortedRows = [];
+
+  if (isASC) {
+    if (index === 4) {
+      sortedRows = Array.from(tableBody.rows).sort((a, b) =>
+        (salaryToNumber(a.cells[index].innerHTML)
+        - salaryToNumber(b.cells[index].innerHTML)));
+    } else {
+      sortedRows = Array.from(tableBody.rows).sort((a, b) =>
+        (a.cells[index].innerHTML.localeCompare(b.cells[index].innerHTML)));
+    }
+  } else {
+    if (index === 4) {
+      sortedRows = Array.from(tableBody.rows).sort((a, b) =>
+        (salaryToNumber(b.cells[index].innerHTML)
+        - salaryToNumber(a.cells[index].innerHTML)));
+    } else {
+      sortedRows = Array.from(tableBody.rows).sort((a, b) =>
+        (b.cells[index].innerHTML.localeCompare(a.cells[index].innerHTML)));
+    }
+  }
+
+  table.tBodies[0].append(...sortedRows);
+}
+
+headers.forEach(function(header, index) {
+  header.addEventListener('click', (e) => {
+    let order;
+
+    if (e.target.dataset.sorted === 'ASC') {
+      e.target.dataset.sorted = 'DESC';
+      order = false;
+    } else {
+      e.target.dataset.sorted = 'ASC';
+      order = true;
+    }
+    sortColumn(index, order);
+  });
+});
+
+const pushNotification = (posTop, posRight, title, description, type) => {
```

Comment:
Why do you use an arrow function here and a function declaration above? Choose something one in order not to confuse other developers